pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -152,6 +152,23 @@ type PropagationSpec struct {
 	// +kubebuilder:validation:Enum=Abort;Overwrite
 	// +optional
 	ConflictResolution ConflictResolution `json:"conflictResolution,omitempty"`
+
+	// ActivationPreference declares when the propagation related content of this Policy will be synced to the Binding.
+	//
+	// The value "Lazy" means the matched resource should delay to sync the propagation related content (e.g: placement) from Policy
+	// to Binding until the resource itself is modified. (In 1:N scenario where one Policy manages many resource templates,
+	// changes to a Policy typically affect numerous applications simultaneously. If there are errors in the Policy
+	// configuration, it could lead to widespread failures. By using this setting to make modified content delay
+	// synchronization from Policy to Binding, the changed Policy can be gradually rolled out through
+	// iterative modifications to resource templates.)
+	//
+	// In a word, The impact of Policy changes on the bondage relationship between resources and Policy is always immediate,
+	// but whether the resource needs to immediately sync the propagation related content to Binding is determined by this field.
+	//
+	// If the field is unset, the default preference is do the sync immediately.
+	//
+	// +optional
```

Comment:
```suggestion
	// ActivationPreference indicates how the referencing resource template will
	// be propagated, in case of policy changes.
	//
	// If empty, the resource template will respond to policy changes
	// immediately, in other words, any policy changes will drive the resource
	// template to be propagated immediately as per the current propagation rules.
	//
	// If the value is 'Lazy' means the policy changes will not take effect for now
	// but defer to the resource template changes, in other words, the resource
	// template will not be propagated as per the current propagation rules until
	// there is an update on it.
	// This is an experimental feature that might help in a scenario where a policy
	// manages huge amount of resource templates, changes to a policy typically
	// affect numerous applications simultaneously. A minor misconfiguration
	// could lead to widespread failures. With this feature, the change can be
	// gradually rolled out through iterative modifications of resource templates.
	//
	// +kubebuilder:validation:Enum=Lazy
	// +optional
```
How this look like? @chaunceyjiang @chaosi-zju

Reply:
lgtm